compose/service.py

```diff
@@ -657,7 +657,7 @@ def _get_container_host_config(self, override_options, one_off=False):
         cap_add = options.get('cap_add', None)
         cap_drop = options.get('cap_drop', None)
         log_config = LogConfig(
-            type=options.get('log_driver', 'json-file'),
+            type=options.get('log_driver', ""),
             config=options.get('log_opt', None)
         )
```

Comment:
That's not what happened. I added one, then removed it and added the test at the docker-py level and at the time, it showed up a docker bug.

https://github.com/docker/docker-py/pull/718/files#diff-347ad10bf29330a41397ea485670f027R285

https://github.com/docker/docker/issues/15633

Which has now been fixed, so we can remove the skip test from docker-py.